Notify/Notify/Notify/AppShell.xaml.cs

```diff
@@ -192,6 +201,41 @@ private List<string> getAllArrivedDestinations(Location location)
             
             return destinationsArrived;
         }
+
+        private List<Notification> getAllElapsedTimeNotifications()
+        {
+            string notificationsJson = Preferences.Get(Constants.PREFERENCES_NOTIFICATIONS, string.Empty);
+            List<Notification> notifications = JsonConvert.DeserializeObject<List<Notification>>(notificationsJson);
+            List<Notification> elapsedTimeNotifications;
+            DateTime notificationTime;
+            
+            r_Logger.LogInformation("Getting all elapsed time notifications");
+            
+            elapsedTimeNotifications = notifications
+                .FindAll(
+                    notification =>
+                    {
+                        bool isTimeNotification = notification.Type is NotificationType.Time;
+                        bool isTimeElapsed = false;
+                        if (notification.TypeInfo is DateTime time)
+                        {
+                            notificationTime = time;
```

Comment:
Do we need "notificationTime" variable?
We can only use the "time" variable from the if-statement for the boolean value right afterwards.
Plus, use a different name for the time of the notification than "time" (notificationTime would be good enough :) )

Reply:
Right, we don't need it. removing it.